plugins/Jobs.cpp

```diff
@@ -638,3 +665,23 @@ string BedrockPlugin_Jobs::_constructNextRunDATETIME(const string& lastScheduled
     // Combine the parts together and return the full DATETIME statement
     return "DATETIME( " + SComposeList(safeParts) + " )";
 }
+
+/**
+ * Returns the number of RUNNING or QUEUED jobs that have the passed jobID as their parent.
+ * @param db
```

Comment:
Why add these when not adding descriptions, convention? I find this useless, nab